doc/platforms.md

```diff
@@ -0,0 +1,28 @@
+# Supported platforms
+
+Flame runs on top of Flutter, so its supported platforms depended on how mature/stable that platform is supported by Flutter.
```

Comment:
Depended -> depend
That platform is supported by -> support for each platform is by